src/OrderLinesList/LineExportSettingModalContainer.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+
+import LineExportSettingModalContainer from './LineExportSettingModalContainer';
+import ExportSettingsModalContainer from '../common/ExportSettingsModal/ExportSettingsModalContainer';
+import { fetchExportDataByIds } from '../common/utils';
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  fetchAllRecords: jest.fn().mockResolvedValue([{ purchaseOrderId: '0610be6d-0ddd-494b-b867-19f63d8b5d6d' }]),
+}));
+
+jest.mock('../common/ExportSettingsModal/ExportSettingsModalContainer', () => jest.fn().mockReturnValue('ExportSettingsModalContainer'));
+
+jest.mock('../common/utils', () => ({
+  ...jest.requireActual('../common/utils'),
+  fetchExportDataByIds: jest.fn(),
+}));
+
+const defaultProps = {
+  linesQuery: '',
+  onCancel: jest.fn(),
+  mutator: {
+    exportLines: {
+      GET: jest.fn(),
+    },
+    exportOrders: {
+      GET: jest.fn(),
+    },
+  },
+};
+
+const renderLineExportSettingModalContainer = (props = {}) => render(
+  <LineExportSettingModalContainer
+    {...defaultProps}
+    {...props}
+  />,
+);
+
+describe('LineExportSettingModalContainer', () => {
+  it('should render Export Settings Modal Container', () => {
+    renderLineExportSettingModalContainer();
+
+    expect(screen.getByText('ExportSettingsModalContainer')).toBeInTheDocument();
+  });
+
+  it('should call fetchAllRecords function if exporting', async () => {
+    renderLineExportSettingModalContainer();
+
+    await act(() => ExportSettingsModalContainer.mock.calls[0][0].fetchOrdersAndLines());
```

Comment:
`await act` looks not so obvious to me too.
If we keep to "arrange-act-assert" pattern, it should be here more like:
- act
- then await for something to appear (or be called)

Reply:
yeah, thanks, I changed it